src/themes/default/components/core/ProductAttribute.vue

```diff
@@ -10,37 +10,34 @@ import ProductAttribute from '@vue-storefront/core/components/ProductAttribute'
 
 export default {
   mixins: [ProductAttribute],
-  data () {
-    return {
-      label: '',
-      value: ''
-    }
-  },
-  beforeMount () {
-    this.label = this.attribute.default_frontend_label
-    let parsedValues = this.product[this.attribute.attribute_code]
-
-    if (!parsedValues) {
-      this.value = this.emptyPlaceholder
-    } else {
-      parsedValues = parsedValues.split(',')
-      let results = []
-      for (let parsedVal of parsedValues) {
-        if (this.attribute.options) {
-          let option = this.attribute.options.find(av => {
-            /* eslint eqeqeq: "off" */
-            return av.value == parsedVal
-          })
-          if (option) {
-            results.push(option.label)
+  computed: {
```

Comment:
Shouldn’t this logic be just in the core? I’m not sure but I think it’s duplicated in the *.vue component is kind of DRY principle violation.
Shouldn’t it be just left in the core?

Reply:
You are absolutelty right, It was originally duplicated - to make sure I'll compare both files and remove duplicated logic if there are no differences.